src/test/java/com/vkatit/cinema/CinemaApplicationTests.java

```diff
@@ -14,9 +28,43 @@ class CinemaApplicationTests {
 	@Autowired
 	ApplicationContext context;
 
+	@Autowired
+	private DownloadController downloadController;
+
+	@Value("${ticket.path}")
+	private String TICKET_PATH;
+
 	@Test
 	void contextLoads() {
 		assertTrue(context != null);
 	}
 
+	@Test
+	@DirtiesContext
```

Comment:
what is the reason of using this annotation?

Reply:
DirtiesContext annotations have been removed since these tests do not pollute the context